### DISCUSSION TITLE
Hover on the autocompletion dropdown list item auto pop up?

### DISCUSSION THREAD

**Discussion by durianwaffle** [eclipse-langium/langium#1648] [Q&A] [answered]
I find that completionItem.documentation need to be **highlighted** and user need to **click** on one of the documentations to show up.
Are there other ways to show the documentation when that completionItem is not highlighted?
**For example, is there any services like hoverProvider that works on completion Item?**
![image](https://github.com/user-attachments/assets/4468b73a-6762-440d-9fde-cfb7c8431a38)

**Comment by msujew** [comment, marked as answer]
Hey @durianwaffle,

I don't think that VS Code supports this. You may want to create a feature request over at [their repository](https://github.com/microsoft/vscode).

**Reply by durianwaffle** [reply]
Thanks for the suggestion! I have submitted a feature request in vscode repo.
https://github.com/microsoft/vscode/issues/226752
